test/jdk/java/lang/StringBuffer/IndexOf.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2000, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2000, 2024 Oracle and/or its affiliates. All rights reserved.
```

Comment:
This copyright header validation failure. Missing comma `,` after 2024.

Reply:
Fixed.